help/Docfiles/BasicProvers.var_wlog_tac.doc

```diff
@@ -0,0 +1,25 @@
+\DOC
+
+\TYPE {term quotation -> term quotation list -> tactic}
+
+\SYNOPSIS
+Assume a proposition without loss of generality.
+
+\DESCRIBE
+Given {var_wlog_then q vars_q} let {asm ⊢? c} be the the goal. {q} is parsed
+in the goal context to a proposition {P}. {vars_q} are parsed to variables in
+the goal context. Let {egv} (effectively free variables) be the free
+variables of {P} and {c} that are not free in the assumptions and are not in
+{vars}, from left to right and first {P}, then {c}. Let {gen_vars} be {vars @
+egv}. Let {asm'} be the elements of {asm} in which any of {vars} is a free
+variable. Let {ant} be the result of splicing {p :: asm'}. The first subgoal
+is {asm, (∀(gen_vars). ant ⇒ c), ¬P ⊢? c}. The proposition {P} is added to the
+assumptios with {strip_assume_tac}. If this generates subgoals (as is usually
+the case), then those subgoals follow.
+
```

Comment:
By the end of this, I still have no idea what this tactic does. The description is too low-level. And some examples would help in the documentation.